docs/src/pages/components/data-grid/rows/rows.md

```diff
@@ -114,9 +114,9 @@ The grid exposes a set of methods that enables all of these features using the i
 
 > ⚠️ Only use this API when you have no alternative. Always start from the declarative API that the grid exposes.
 
-- `getSortModel`: Get the sort model currently applied in the grid.
-- `setSortModel`: Set the sort model of the component and trigger a new sorting of rows.
-- `onSortModelChange`: Callback fired when the columns sorting changed before the grid has sorted its rows.
+- `getSortModel`: Get the sort model currently applied to the grid.
+- `setSortModel`: Set the sort model and trigger the sorting of rows.
+- `onSortModelChange`: Callback fired when the column sorting changed before the grid has sorted its rows.
```

Comment:
"before the grid has sorted its rows"

Is this referring to row sort order, or column sort order within rows?

Reply:
What's the difference?

I think that we should remove the reference to this. There is one state, it controls how the header communicate to the users the sort order and how the rows are sorted. This state is displayed to the user in a best effort way. Sorting the rows can take a bit of time.

```suggestion
	- `onSortModelChange`: Callback fired when the column sorting changes.
```

Reply:
> What's the difference?

One is rows, the other is columns, so quite a significant difference!

Reply:
I'm confused, it's the same?

Reply:
The description is ambiguous, but if the prop is being removed, it doesn't matter.

Reply:
I don't think that we should remove this prop.